integration/airflow/openlineage/lineage_backend/__init__.py

```diff
@@ -148,6 +80,9 @@ class OpenLineageBackend(LineageBackend):
 
     @classmethod
     def send_lineage(cls, *args, **kwargs):
+        # Do not use LineageBackend approach when we can use plugins
```

Comment:
I think this make sense, but is this a reasonable expectation for the user? That is, if I as a user configure the `LineageBackend`, but the plugin get's configured without me knowing, that might be confusing.

Can we log a warning instead? I.e. recommending users to use the plugins based on the version of Airflow they are running?

Reply:
Plugin's functionality is a superset of what LineageBackend can do, so I don't really see use case for that 🙂 

We're not turning plugin off when LineageBackend is configured, so this would result in duplicate events emitted - which I had for a bit while testing.

Reply:
Makes sense, but should we still log a `info` or `warn` message to the user that the plugin will be used to emit events instead of the backend?

That is:
```
if parse_version(AIRFLOW_VERSION) >= parse_version("2.3.0.dev0"):
            return
```

returns on Airflow `>=` `2.3.0`, so I would still let the user know that the plugin will be used instead and maybe recommend that the user disable the backend because it's beed deprecated.